bucket/lockhunter.json

```diff
@@ -14,20 +14,8 @@
         "} Else {",
         "   Get-ChildItem \"$dir\" '*,2.*' | Rename-Item -Path { $_.Fullname } -NewName { $_.Basename.TrimEnd(',2') + $_.Extension }",
         "}",
-        "$url = \"https://raw.githubusercontent.com/ScoopInstaller/Extras/master/scripts/lockhunter/install-context.reg\"",
-        "do_dl $url \"$dir\\install-context.reg\"",
-        "$ok, $err = check_hash \"$dir\\install-context.reg\" \"049a1207f1ca58c93e3880363a7ed9320c0ee68303a659fb3a4ed9299bd7737d\" $(show_app $app $bucket)",
-        "if(!$ok) {",
-        "   error $err",
-        "   abort $(new_issue_msg $app $bucket \"hash check failed\")",
-        "}",
-        "$url = \"https://raw.githubusercontent.com/ScoopInstaller/Extras/master/scripts/lockhunter/uninstall-context.reg\"",
-        "do_dl $url \"$dir\\uninstall-context.reg\"",
-        "$ok, $err = check_hash \"$dir\\uninstall-context.reg\" \"38443a27b65dda6554a8470261064bc312b202ed81c356ef363b10b089aa5790\" $(show_app $app $bucket)",
-        "if(!$ok) {",
-        "   error $err",
-        "   abort $(new_issue_msg $app $bucket \"hash check failed\")",
-        "}"
```

Comment:
`$scoopdir\\scoop\\buckets` can be shortened to `$bucketsdir`

Reply:
Done, also cleaned up a few commits.

Reply:
@rasa requesting a re-review

Reply:
@issaclin32 @rasa can one of you review this, I can confirm it works locally. But I'm not sure why I don't see the github actions check on any commit for this PR anywhere.